### DISCUSSION TITLE
Azure OpenAI

### DISCUSSION THREAD

**Discussion by C45513-L** [sinaptik-ai/pandas-ai#110] [General]
Hi, Im using the Azure OpenaAI and Ive got the error when I do the pandas_ai.run(), is pandas-ai supporting Azure OpenAI? Thanks
InvalidRequestError: Must provide an 'engine' or 'deployment_id' parameter to create a <class 'openai.api_resources.chat_completion.ChatCompletion'>

**Comment by mspronesti** [comment]
Hi @C45513-L ,
I opened a PR yesterday to support Microsoft Azure OpenAI API. It has not been merged yet so this functionality is not supported at the moment.
